SU-SWS/saa_victoria_subtheme
# ADAPT-526: Added mixin for small text size and fixed vivid red banner…

… small text.

# READY FOR REVIEW

# Summary
- Added mixin for a font size that is between modular typography - used for faux-cards and for the vivid red banner small text.

# Review By (Date)
- 7/28

# Urgency
- medium

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. check faux card small text and vivid red banner small text, should both be 17px on desktop.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
